dags/resources/stages/parse/table_definitions/angle/StableMasterFront_call_mint.json

```diff
@@ -0,0 +1,62 @@
+{
+    "parser": {
+        "abi": {
+            "inputs": [
+                {
+                    "internalType": "uint256",
+                    "name": "amount",
+                    "type": "uint256"
+                },
+                {
+                    "internalType": "address",
+                    "name": "user",
+                    "type": "address"
+                },
+                {
+                    "internalType": "contract IPoolManager",
+                    "name": "poolManager",
+                    "type": "address"
+                },
+                {
+                    "internalType": "uint256",
+                    "name": "minStableAmount",
+                    "type": "uint256"
+                }
+            ],
+            "name": "mint",
+            "outputs": [],
+            "stateMutability": "nonpayable",
+            "type": "function"
+        },
+        "contract_address": "0x5addc89785d75c86ab939e9e15bfbbb7fc086a87",
+        "field_mapping": {},
+        "type": "trace"
+    },
+    "table": {
+        "dataset_name": "angle",
+        "schema": [
+            {
+                "description": "",
+                "name": "amount",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "user",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "poolManager",
+                "type": "STRING"
+            },
+            {
+                "description": "",
+                "name": "minStableAmount",
+                "type": "STRING"
+            }
+        ],
+        "table_description": "",
+        "table_name": "StableMasterFront_call_event_mint"
```

Comment:
Should be StableMasterFront_call_mint

Reply:
fixed. Check still fails on contract address field: 
SELECT '0x5efe48f8383921d950683c46b87e28e21dea9fb5' UNION ALL SELECT '0x98fdbc5497599eff830923ea1ee152adb9a4cea5' UNION ALL SELECT '0x4121a258674e507c990cdf390f74d4ef27592114' UNION ALL SELECT '0xb924497a1157b1f8835c93cb7f3d4aa6d2f227ba' UNION ALL SELECT '0xfc8f9eefc5fce1d9dace2b0a11a1e184381787c4'

Reply:
What if you use this syntax instead: https://github.com/blockchain-etl/ethereum-etl-airflow/blob/425ebdbb573fb4db2842ede571ef79f3f4d07298/dags/resources/stages/parse/table_definitions/synthetix/sBTC_event_SelfDestructTerminated.json#L9

Reply:
Worked! thank you @araa47